app/lib/cost_calculator.rb

```diff
@@ -0,0 +1,28 @@
+module CostCalculator
+  class << self
+    def cost(type, object, vat)
+      case type
+      when :travel_and_waiting_total
+        travel_and_waiting_total(vat, object)
+      end
```

Comment:
**nit** could simplify until other "cases" come up

```ruby
travel_and_waiting_total(vat, object) if type == :travel_and_waiting_total
```

Reply:
going to extend this in my next ticket so want to leave it for now